jvm-runtime/ftl-runtime/.maven_settings.xml

```diff
@@ -1,14 +1,9 @@
 <settings>
     <servers>
         <server>
-            <id>ossrh-snapshots</id>
-            <username>${env.SONATYPE_USERNAME}</username>
-            <password>${env.SONATYPE_PASSWORD}</password>
-        </server>
-        <server>
-            <id>ossrh-releases</id>
-            <username>${env.SONATYPE_USERNAME}</username>
-            <password>${env.SONATYPE_PASSWORD}</password>
+            <id>central</id>
```

Comment:
You've exported these in the GH Actions workflow as "SONATYPE_USERNAME" and "SONATYPE_PASSWORD".